studyzen/Lectures/Requests/UpdateLectureRequest.cs

```diff
@@ -0,0 +1,12 @@
+namespace StudyZen.Lectures.Requests;
+
+public sealed class UpdateLectureRequest
+{
+    public string? Name { get; }
```

Comment:
Make strings of the class non nullable, however allow passing nullable arguments in constructor. Call extension method `x= x.ThrowIfArgumentNull()`. There is nothing stopping me from passing a request such as

```
{
  "name": null,
  "description": null
}
```

Reply:
i guess because with patch request not necessarily  all fields are updated?

Reply:
oh yeah true it's patch